parts/kubernetesmastercustomscript.sh

```diff
@@ -186,7 +186,7 @@ function configAzureNetworkPolicy() {
     mkdir -p $CNI_BIN_DIR
 
     # Mirror from https://github.com/Azure/azure-container-networking/releases/tag/$AZURE_PLUGIN_VER/azure-vnet-cni-linux-amd64-$AZURE_PLUGIN_VER.tgz
-    downloadUrl https://acs-mirror.azureedge.net/cni/azure-vnet-cni-linux-amd64-latest.tgz | tar -xz -C $CNI_BIN_DIR
+    downloadUrl https://github.com/ofiliz/hello/releases/download/pre-release/azure-vnet-cni-linux-amd64-v0.8-7-g8acef7d.tgz | tar -xz -C $CNI_BIN_DIR
```

Comment:
I don't think we don't want to merge this to master until we test/vet the external `azure-vnet-cni-linux-amd64` download and then host it in `https://acs-mirror.azureedge.net/cni`.